test/utilities/StyleMocking.js

```diff
@@ -0,0 +1,62 @@
+import assert from 'assert'
+
+// These methods are useful in tests which verify changes a method makes to element 'style'
+// properties.
+
+/**
+ * An object whose properties are taken to be style property names,
+ * and whose property values are taken to be style property values.
+ * @typedef {Object} StylesHash
```

Comment:
Let's keep our eyes peeled when looking at other repos and maybe we'll stumble upon it one day!